cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/RightsizeResult.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.servlet.response;
+
+import com.google.gson.Gson;
+import com.linkedin.cruisecontrol.servlet.parameters.CruiseControlParameters;
+import com.linkedin.kafka.cruisecontrol.config.KafkaCruiseControlConfig;
+import com.linkedin.kafka.cruisecontrol.detector.ProvisionerState;
+import java.util.HashMap;
+import java.util.Map;
+
+
+
+@JsonResponseClass
+public class RightsizeResult extends AbstractCruiseControlResponse {
+  @JsonResponseField
+  protected static final String BROKER_DIFF = "broker-diff";
+  @JsonResponseField
+  protected static final String PARTITION_COUNT = "partition-count";
+  @JsonResponseField
+  protected static final String TOPIC_NAME = "topic-name";
```

Comment:
Should we `TOPIC_NAME = "topic-name";` -> `TOPIC = "topic";` and make the relevant param name changes in yaml file?